src/app/pages/news/11.2/04-engine-cockpit.md

```diff
@@ -1,6 +1,13 @@
 ## Engine Cockpit {#engineCockpit}
 
-Blah Engine Cockpit blah
+We added awesome new world-class monitoring, user management, operations, and configuration features to the Engine Cockpit.
+
+- **Configuration**: Use the new **System/SSL** view to manage your certificates. The new **System/Config File Editor** helps you view and edit any configuration file of Axon Ivy. Press CTRL-SPACE to get content assistance. 
```

Comment:
Ok to start with, the second part will get a distinct news entry  ... so I'll consider to shorten it here; and link it to the news section below